test/util/comparison.coffee

```diff
@@ -52,11 +52,18 @@ describe 'equals', ->
     equals(new Foo('abc', [1,2,3]), new Foo('abc', [1,2,3])).should.be.true()
     equals(new Foo('abc', [1,2,3]), new Foo('abcd', [1,2,3])).should.be.false()
     equals(new Foo('abc', new Bar('xyz', [1,2,3])), new Foo('abc', new Bar('xyz', [1,2,3]))).should.be.true()
-    equals(new Foo('abc', new Bar('xyz')), new Foo('abc', new Bar('xyz'))).should.be.false() # because Bar is missing components
-    equals(new Foo('abc', new Bar('xyz')), new Foo('abc', new Bar('xyz',999))).should.be.false()
+    should.not.exist(equals(new Foo('abc', new Bar('xyz')), new Foo('abc', new Bar('xyz')))) # because Bar is missing components
```

Comment:
`should.not.exist(result)` may allow `undefined`. Perhaps use `should(result).be.null()` to match other tests in the project that check for null.